### PR TITLE
[Merged by Bors] - Add Run a Node guide

### PR DESCRIPTION
**Pull request by kevinbogner**
## Issue Addressed

Related to #3672  

## Proposed Changes

- Added a guide to run a node. Mainly, copy and paste from 'Merge Migration' and 'Checkpoint Sync'.
- Ranked it high in ToC:
  - Introduction
  - Installation
  - Run a Node
  - Become a Validator
	...
- Hid 'Merge Migration' in ToC.

## Additional Info

- Should I add/rephrase/delete something?
- Now there is some redundancy:
  - 'Run a node' and 'Checkpoint Sync' contain similar information.
  - Same for 'Run a node' and 'Become a Validator'.

### CODE REVIEW

**Comment by michaelsproul** [comment]
github was being super flaky all day so I didn't get a chance to review this yet

will get onto it soon!

File: book/src/run_a_node.md
Comment (michaelsproul) [inline book/src/run_a_node.md]: Rather than duplicating the FAQ I think we can link to the FAQs on the merge migration and checkpoint sync pages

File: book/src/run_a_node.md
Comment (michaelsproul) [inline book/src/run_a_node.md]: I think we could break this down into more granular steps like:

1. Choose a checkpoint sync provider (describe just how to choose the URL)
2. Set up an execution node (link to EL docs, etc)
3. Run Lighthouse
4. Check logs (example of `INFO Starting checkpoint sync` and `INFO Synced`)
5. Further reading (links to other sections like Becoming a Validator, Validator Monitor, Checkpoint Sync, Merge-Migration, HTTP API, etc).

In step 3 we show an example of the full command-line used for a staking node and a non-staking node respectively, e.g.

Staking:

```bash
lighthouse bn \
  --http \
  --checkpoint-sync-url https://mainnet.checkpoint.sigp.io \
  --execution-endpoint http://localhost:8551 \
  --execution-jwt /secrets/jwt.hex 
```

Non-staking:

```bash
lighthouse bn \
  --checkpoint-sync-url https://mainnet.checkpoint.sigp.io \
  --execution-endpoint http://localhost:8551 \
  --execution-jwt /secrets/jwt.hex \
  --disable-deposit-contract-sync
```

**Review by michaelsproul — COMMENTED** [review]
This is looking structurally very nice!

I had a bit more of a radical overhaul in mind, if you have time to implement some of those changes that would be awesome. It doesn't need to be 100% perfect though, and we can always refine it incrementally after merging.

Thanks again!

**Comment by michaelsproul** [comment]
Oh, would also be cool to mention the `--network` flag so users can choose a network.

**Comment by kevinbogner** [comment]
A radical overhaul sounds excellent haha. 

I will probably be done early next week. I will ping you on dc to let you know it's ready for review

File: book/src/SUMMARY.md
Comment (kevinbogner) [inline book/src/SUMMARY.md:49]: Added 'Merge Migration' to the 'Advanced Usage' section. Not sure if this is the right place for it. What do you think?

File: book/src/run_a_node.md
Comment (kevinbogner) [inline book/src/run_a_node.md:108]: Is the 'Check logs' information I provide enough? Is there anything I could add or delete?
I'm struggling the most to apply appropriate information for this section

File: book/src/run_a_node.md
Comment (kevinbogner) [inline book/src/run_a_node.md]: Is 'Prater' still the current testnet?

**Review by kevinbogner — COMMENTED** [review]
Added your suggested steps.

File: book/src/SUMMARY.md
Comment (michaelsproul) [reply book/src/SUMMARY.md:49]: I think that's good!

File: book/src/run_a_node.md
Comment (michaelsproul) [inline book/src/run_a_node.md]:
```suggestion
Checkpoint sync is *optional*; however, we recommend it since it is substantially faster
```

File: book/src/run_a_node.md
Comment (michaelsproul) [inline book/src/run_a_node.md]:
```suggestion
Lighthouse supports fast sync from a recent finalized checkpoint.
```

File: book/src/run_a_node.md
Comment (michaelsproul) [inline book/src/run_a_node.md]:
```suggestion
For example, the URL for Sigma Prime's checkpoint sync server for mainnet is `https://mainnet.checkpoint.sigp.io`,
```

File: book/src/run_a_node.md
Comment (michaelsproul) [inline book/src/run_a_node.md]:
```suggestion
For now, we will copy the URL to the clipboard.
```

I think lets streamline this to recommend using a community endpoint, and add an aside about syncing from an existing node.

File: book/src/run_a_node.md
Comment (michaelsproul) [inline book/src/run_a_node.md]: Let's delete this section from here and move the checkpoint sync security check to the `Check logs` section.

File: book/src/run_a_node.md
Comment (michaelsproul) [inline book/src/run_a_node.md:31]: Let's add Erigon to this section (and the merge migration docs). I think the go-to URL is https://github.com/ledgerwatch/erigon#beacon-chain-consensus-layer

File: book/src/run_a_node.md
Comment (michaelsproul) [reply book/src/run_a_node.md]: Kind of. It merged with Goerli and we support both names as an alias for the same thing. I think in these user-facing docs we may as well call it Goerli and link to the Goerli homepage: https://goerli.net/

The other long-lived testnet is Sepolia, which I think we should list too (https://sepolia.dev/).

And for completeness, Gnosis chain (https://www.gnosis.io/), which is a side-chain using Ethereum consensus.

File: book/src/run_a_node.md
Comment (michaelsproul) [inline book/src/run_a_node.md]: Let's add a sentence here about why the HTTP flag is needed:

> The HTTP API is required for the beacon node to accept connections from the _validator _client_, which manages keys.

File: book/src/run_a_node.md
Comment (michaelsproul) [inline book/src/run_a_node.md]:
```suggestion
Check out the [FAQ](./checkpoint-sync.md#faq) for more information on checkpoint sync.
```

File: book/src/run_a_node.md
Comment (michaelsproul) [inline book/src/run_a_node.md]: I'm torn as to whether we should swap the checkpoint sync and execution node sections, seeing as we talk about copying a URL to the clipboard.

File: book/src/run_a_node.md
Comment (michaelsproul) [inline book/src/run_a_node.md:123]: This is where we should add a note about checking the checkpoint against one of the providers from https://eth-clients.github.io/checkpoint-sync-endpoints/, with another aside about "or a friend's node/block explorer"

**Review by michaelsproul — COMMENTED** [review]
This is really great, thank you :clap:

I'm excited to get this merged!

File: book/src/run_a_node.md
Comment (michaelsproul) [reply book/src/run_a_node.md:108]: I think it's sufficient!

**Comment by kevinbogner** [comment]
Thanks :) I made some changes:
- Committed your suggestions
- Added Erigon to 'Run a Node' and 'Merge Migration' section
- Added the security note to check against a trusted source to the 'Check logs' section
- Changed testnet to Goerli and added the testnets for Sepolia and Gnosis chain. 
- Switched step 1 with step 2. I think it is more straightforward now. I also deleted the 'clipboard sentence'. It was not ideal.

Let me know if there's anything else we should change.

Also saw a few checks failed. Maybe you can look into them and help me to understand what caused them. https://github.com/sigp/lighthouse/actions/runs/3444425151/jobs/5952589728 should be fixed now.

File: book/src/run_a_node.md
Comment (michaelsproul) [inline book/src/run_a_node.md]:
```suggestion
You should be finished with one [Installation](./installation.md) method of your choice to continue with the following steps:
```

File: book/src/run_a_node.md
Comment (michaelsproul) [inline book/src/run_a_node.md]:
```suggestion
than syncing from genesis while still providing the same functionality.
```

**Review by michaelsproul — APPROVED** [review]
Approved modulo two minor tweaks to wording.

I'll merge now!

**Pull request closed without merging**
